src/io/fs/fs.cc

```diff
@@ -37,8 +37,13 @@ void frontend::moveFile(const std::string& originFilename, const std::string& ne
     system(command.c_str());
 }
 
-void frontend::copyFile(const std::string& originFilename, const std::string& newFilename) {
-    std::string command(fmt::format("cp -r {} {}", originFilename, newFilename));
+void frontend::copyFile(const std::string& originFilename, const std::string& newFilename, const bool withDir) {
+    std::string command;
+    if (withDir) {
+        command = fmt::format("cp -r {} {}", originFilename, newFilename);
+    } else {
+        command = fmt::format("cp {} {}", originFilename, newFilename);
+    }
```

Comment:
I wonder for what reason.

Reply:
Not always we want to copy with dir.
`frontend::copyFile(FLAGS_install, fmt::format("{}extension", frontend::CurrentPath::get()), false);`
copying the hole dir is not allowed or useless here.